dataset/dictionary/enemies.json5

```diff
@@ -959,52 +1119,69 @@
   {
     en: "Rifthound Whelp",
     ja: "獣域ウェルプ",
+    zh: "兽境幼兽",
     pronunciationJa: "じゅういきウェルプ",
+    variants: {
+      zh: [ "小流血狗" ],
+    },
     notes: "whelp はイヌ科の動物の子供を指す語。",
     tags: [ "enemy" ],
   },
   {
     en: "Rockfond Rifthound Whelp",
     ja: "獣域ウェルプ・岩",
+    zh: "兽境幼兽·岩",
     pronunciationJa: "じゅういきウェルプ・いわ",
     notes: "whelp はイヌ科の動物の子供を指す語。",
     tags: [ "enemy" ],
   },
   {
     en: "Thundercraven Rifthound Whelp",
     ja: "獣域ウェルプ・雷",
+    zh: "兽境幼兽·雷",
     pronunciationJa: "じゅういきウェルプ・かみなり",
     notes: "whelp は若いイヌ科の動物を指す語。",
     tags: [ "enemy" ],
   },
   {
     en: "Rifthound",
     ja: "獣域ハウンド",
+    zh: "兽境猎犬",
     pronunciationJa: "じゅういきハウンド",
+    variants: {
+      zh: [ "流血狗", "大流血狗" ],
+    },
     tags: [ "enemy" ],
   },
   {
     en: "Rockfond Rifthound",
     ja: "獣域ハウンド・岩",
+    zh: "兽境猎犬·岩",
     pronunciationJa: "じゅういきハウンド・いわ",
     tags: [ "enemy" ],
   },
   {
     en: "Thundercraven Rifthound",
     ja: "獣域ハウンド・雷",
+    zh: "兽境猎犬·雷",
```

Comment:
This seems wrong translation.

![](https://user-images.githubusercontent.com/107226297/174476687-596e224f-d892-4307-b0bb-37dbe7339bda.png)